src/Test/PdbUtilities/Roslyn.Test.PdbUtilities.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Roslyn.Test.PdbUtilities</RootNamespace>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>$(RoslynLibraryTargetFrameworks)</TargetFrameworks>
```

Comment:
This change means that when you build for `netcoreapp3.1` then yes you get the single TF build experience. But when you build for `net472` you still build all TF. Why did we not change this to multi-target between `net472` and `netcoreapp3.1` instead?

Reply:
MS.CA.CSharp.Test.Utilities and MS.CA.VisualBasic.Test.Utilities depend on this. We could make those target 'netcoreapp3.1;net472' as well. Let me know if I had the right idea in 56f5b10.

Reply:
I would go ahead and move those to `netcoreapp3.1;net472`. If we're going to do this lets commit to it and do it right.